test/provisioner/provision_ibmcloud_initializer.go

```diff
@@ -176,11 +176,13 @@ func initProperties(properties map[string]string) error {
 		if len(IBMCloudProps.Region) <= 0 {
 			return errors.New("REGION was not set.")
 		}
-		if len(IBMCloudProps.KubeVersion) <= 0 {
-			return errors.New("KUBE_VERSION was not set, get it via command: ibmcloud cs versions")
-		}
-		if len(IBMCloudProps.WorkerOS) <= 0 {
-			return errors.New("WORKER_OPERATION_SYSTEM was not set, set it like: UBUNTU_20_64, UBUNTU_18_S390X")
+		if Action != "uploadimage" {
```

Comment:
I would prefer for us to re-think this code and remove validation for the initializer and instead create some sort of runners interface with implementations for provision cluster, install, upload image, uninstall, deprovision cluster that would have (at least) a validate and 'run' function that could do the checks. However that is a big change, so I think this is fine for now.